charts/cmc-claim-store/values.yaml

```diff
@@ -43,4 +43,4 @@ java:
     RPA_NOTIFICATIONS_RESPONSERECIPIENT: civilmoneyclaims+rpa-defence-response@gmail.com
     RPA_NOTIFICATIONS_COUNTYCOURTJUDGEMENTRECIPIENT: civilmoneyclaims+rpa-county-court-judgement@gmail.com
     RPA_NOTIFICATIONS_PAIDINFULLRECIPIENT: civilmoneyclaims+rpa-paid-in-full@gmail.com
-    DOCUMENT_MANAGEMENT_USERROLES: caseworker-cmc
+    DOCUMENT_MANAGEMENT_USERROLES: caseworker-cmc,citizen
```

Comment:
you might want to update the environment.properties as well

Reply:
thanks for reminding, i'had to take them out